misc/de-metas-common/de-metas-common-product/src/main/java/de/metas/common/product/v2/response/JsonProduct.java

```diff
@@ -103,15 +105,24 @@
 	@Nullable
 	@JsonProperty("manufacturerName")
 	String manufacturerName;
-	
+
 	@ApiModelProperty( //
 			allowEmptyValue = true, //
 			dataType = "java.lang.String", //
 			value = "This translates to `M_Product.ManufacturerArticleNumber`.")
 	@Nullable
 	@JsonProperty("manufacturerNumber")
 	String manufacturerNumber;
-	
+
+	@ApiModelProperty( //
+			allowEmptyValue = true, //
```

Comment:
are we sure it makes sense to allow an empty value here? what does "empty" mean here?
note that empty is different from null..

Reply:
right, we need to remove that 👍

Reply:
removed